client/core/bond.go

```diff
@@ -672,54 +708,193 @@ func (c *Core) UpdateBondOptions(form *BondOptionsForm) error {
 	if err != nil {
 		return err
 	}
+	// TODO: exclude unregistered and/or watch-only
 	acct, err := c.db.Account(form.Addr)
 	if err != nil {
 		return err
 	}
+
+	bondAssets, _ := dc.bondAssets()
+	if bondAssets == nil {
+		return errors.New("dex not connected")
```

Comment:
Just remembered I cannot do this otherwise user can't modify options (thus reserves) if the host is down.  Just need to update the two places below to do the nil check.